DigitalLearningSolutions.Web/ViewComponents/InsetTextViewComponent.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace DigitalLearningSolutions.Web.ViewComponents
+{
+    using DigitalLearningSolutions.Web.ViewModels.Common.ViewComponents;
+    using Microsoft.AspNetCore.Mvc;
+
+    public class InsetTextViewComponent : ViewComponent
+    {
+        public IViewComponentResult Invoke(
+            string text,
+            string cssClass

```

Comment:
cssClassString?
Sounds a bit like you're only allowed to pass one in. Feel free to merge after that.

Reply:
This is the same parameter naming as every other view component that has a css class parameter

Reply:
Ah. Fair, let's leave it as-is then.